ui/app/appointments/constants.js

```diff
@@ -26,7 +26,8 @@ Bahmni.Appointments.Constants = (function () {
         appointmentStatusList: [ "Scheduled", "CheckedIn", "Completed", "Cancelled", "Missed" ],
         regexForTime: /^(?:(?:1[0-2]|0?[1-9]):[0-5]\d\s*[AaPp][Mm])?$/,
         privilegeManageAppointments: 'app:appointments:manageAppointmentsTab',
-        privilegeForAdmin: 'app:appointments:adminTab'
+        privilegeForAdmin: 'app:appointments:adminTab',
+        availableForAppointments: 'Available for appointments'
```

Comment:
This is a provide attribute to be defined in OpenMRS. Does the appointments omod create this attribute as part of migration script?

Reply:
Yes